tools/corrade.py

```diff
@@ -25,10 +25,16 @@ def find_in_string(data, text):
     if conf.env.CXX_NAME in ["gcc", "g++"] and int(conf.env['CC_VERSION'][0]+conf.env['CC_VERSION'][1]) < 48:
         conf.fatal('Corrade cannot be setup with GCC < 4.8!')
 
+
     includes_check = ['/usr/local/include', '/usr/include', '/opt/local/include', '/sw/include']
     libs_check = ['/usr/lib', '/usr/local/lib', '/opt/local/lib', '/sw/lib', '/lib', '/usr/lib/x86_64-linux-gnu/', '/usr/lib64']
     bins_check = ['/usr/bin', '/usr/local/bin', '/opt/local/bin', '/sw/bin', '/bin']
 
+    if conf.options.corrade_install_dir:
```

Comment:
I think when the user specifies the directory, it doesn't make sense to keep the other hints. The compiler might also be confused because it can find it in multiple directories..

Reply:
Fair enough. Will fix.